tools/dragen-germline/4.3.6/dragen-germline__4.3.6.cwl

```diff
@@ -825,6 +841,28 @@ inputs:
     inputBinding:
       prefix: "--hla-min-reads="
       separate: False
+
+  # Multi-Region Joint Detection
+  enable_mrjd:
+    label: enable multi-region joint detection
+    doc: |
+      In DRAGEN v4.3, MRJD covers regions that include six clinically relevant genes: NEB, TTN, SMN1/2, PMS2, STRC, and IKBKG.
+      With this option enabled, the following two types of variants are reported: 1. Uniquely placed variants; 2. Region-ambiguous variants.
+    type: boolean?
+    inputBinding:
```

Comment:
At the moment, you have `--enable-mrjd` available in the input binding, which means it will be part of the main dragen command cli args (which is not what you want).

Reply:
In CWL, the inputBinding attribute on an inputs is not a mandatory field